gluten-data/src/main/scala/io/glutenproject/exec/ExecutionCtx.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.glutenproject.exec
+
+import io.glutenproject.init.BackendJniWrapper
+
+import org.apache.spark.util.TaskResource
+
+class ExecutionCtx extends TaskResource {
+  private val handle = BackendJniWrapper.createExecutionCtx()
```

Comment:
It seems this is a dead code, we did not have `BackendJniWrapper_createExecutionCtx` at jni side. Can you describe how/when to use this new class ? If it is just a part of future requirement, I prefer add this code when using it.

Reply:
This patch does not change any functionality, and ExecutionCtx should be added in next patch as you said, thanks!

Reply:
already revert.